content/docs/introduction/glossary.md

```diff
@@ -51,6 +51,12 @@ metrics that are exposed in a non-Prometheus format into a format Prometheus sup
 
 An instance is a label that uniquely identifies a target in a job.
 
+### Label
+
+A label is a key-value pair associated with a metric. Labels allow for identification of different instances of the same metric.
```

Comment:
instance has a meaning in Prometheus, so I'd go for another word.

Reply:
Occurence?

Reply:
Occurrence could be misunderstood to have a time aspect. Variation maybe?

Hmm, this definition makes it seem like there's only one label for a metric. And in Prometheus terms we should be talking about time series rather than metrics here.

Reply:
We typically define "metric" as a unique time-series. This would also be good to add to the glossary term list.

Reply:
That's not what a metric is in Prometheus, a metric can contain any number of time series.

Reply:
That would be a metric family… =:-o
We've always been lacking complete consistency here.

Reply:
Yeah, which is why it's best to avoid "metric".

Reply:
Indeed. (But that's a different reason for not using metric than what you said above.)

(In different news, perhaps we should one day officially define those terms for Prometheus instead of avoiding them. IIRC, we deferred the discussion at some point in the past, and then forgot about it.)

Reply:
Metric is ambiguous here, though it does apply to all uses of it.

(I'm not sure we'll get to fix this, I put a bit in my book about all the meanings of metric).

Reply:
One reason I want this to be clarified is the confusion I had when I was starting with Prometheus: time series/metric concept and the fact a time series is identified by its "metric name" but a metric can be a _set_ of time series.

Reply:
As @brian-brazil said, for the purpose of this PR, I would avoid using "metric" as a meaningful term.
Establishing a canonical meaning for "metric" is a different issue and might not be feasible at all.